module/zfs/zio.c

```diff
@@ -2584,17 +2584,24 @@ int
 zio_resume(spa_t *spa)
 {
 	zio_t *pio;
+	zio_suspend_reason_t reason;
 
 	/*
 	 * Reexecute all previously suspended i/o.
 	 */
 	mutex_enter(&spa->spa_suspend_lock);
+	reason = spa->spa_suspended;
 	spa->spa_suspended = ZIO_SUSPEND_NONE;
 	cv_broadcast(&spa->spa_suspend_cv);
 	pio = spa->spa_suspend_zio_root;
 	spa->spa_suspend_zio_root = NULL;
 	mutex_exit(&spa->spa_suspend_lock);
 
+	if (reason != ZIO_SUSPEND_NONE)
+		cmn_err(CE_WARN, "Pool '%s' was suspended and is being "
```

Comment:
I think we want to improve the wording here.
When this message is emitted, the pool has been suspended for a while, and "Pool '%s' was suspended" reads like it is happening just now

maybe:
> Pool '%s' is resuming from a suspended state. Failed I/O will be retried.

Reply:
Oh yeah, now I can't unsee it. I've taken your text as-is, thank you!